newscoop/library/Newscoop/Services/TemplatesService.php

```diff
@@ -66,5 +130,11 @@ private function preconfigureSmarty()
 
         // reverse templates dir order
         $this->smarty->setTemplateDir(array_reverse($this->smarty->getTemplateDir()));
+
+        $publicationMetadata = $this->publicationService->getPublicationMetadata();
```

Comment:
what if the publication metadata are empty ? Can be possibility that alias is not set, thus metadata data wont be set